src/ui/FileViewer/index.tsx

```diff
@@ -155,6 +170,21 @@ export default function FileViewer({
   onClose,
   onDelete,
 }: FileViewerProps): ReactElement {
+  // check if the message is single/multi
+  if (message?.isMultipleFilesMessage()) {
+    return (
+      <FileViewerComponent
+        profileUrl={message?.sender?.profileUrl}
+        nickname={message?.sender?.nickname}
+        viewerType={ViewerTypes.MULTI}
```

Comment:
So this part where customer has to give ViewerTypes.MULTI is what I don't like becuase customer would expect when giving fileInfoList would make this for MultiFilesViewer automatically.

Reply:
yeah, you can rewrite that way too